### ISSUE TITLE
輸入網頁代碼時容易出現缺漏

### ISSUE THREAD

**Issue by a19901201** [zhangrenyang/chatgpt#5]
例如我輸入正確的代碼送出後
最後一個字會消失

輸入的代碼
```
                        <CloseIcon style={{ fontSize: '1.2rem' }} />
                      </IconButton>
                    )}
                  </div>
                </ListSubheader>
              }
            >
              {SidebarData[0].menuItems.map(renderMenuItem)}
              <ListItemButton onClick={handleLogout} className="sidebar-logout logout-button" style={{ backgroundColor: '#2F0000' }}>
                <ListItemText primary="登出" style={{ textAlign: 'center' }} />
              </ListItemButton>
            </List>
          </div>
        </Drawer>
      )}
    </Fragment>
  )
}

export default Sidebar
```

發送給GPT的
```
                    <CloseIcon style={{ fontSize: '1.2rem' }} /
                  </IconButton
                )
              </div
            </ListSubheader
          
        
          {SidebarData[0].menuItems.map(renderMenuItem)
          <ListItemButton onClick={handleLogout} className="sidebar-logout logout-button" style={{ backgroundColor: '#2F0000' }}
            <ListItemText primary="登出" style={{ textAlign: 'center' }} /
          </ListItemButton
        </List
      </div
    </Drawer
  )
</Fragment
export default Sideba
```

**Comment by zhangrenyang** [comment]
I have fixed this issue. Please uninstall and reinstall the [ latest version](https://marketplace.visualstudio.com/items?itemName=zhang-renyang.chat-gpt) to retry. 
If you still encounter any problems, please provide feedback again. 
If the issue has been resolved, please close this ticket.

**Comment by a19901201** [comment]
Dear @zhangrenyang,

Thank you so much for fixing the issue! I'm delighted to know that you have resolved the problem of the last character disappearing. Your prompt response and ability to address the issue are truly impressive.

I have uninstalled the previous version and reinstalled the latest version. After testing it, I can confirm that the issue has been completely resolved. I have reviewed the entire code again, and there are no problems anymore. The last character is now displaying correctly.

Wishing you all the best!
